### ISSUE TITLE
打扰了

### ISSUE THREAD

**Issue by zycc853** [MasLiang/Learning-without-Forgetting-using-Pytorch#3]
您好,请问您是不是只比较了一次,就是新旧任务进行比较,请问您有做连续十个任务那种的比较方式吗?

**Comment by MasLiang** [comment]
In the 'readme.md', I gave some experiments. Although there is not a case that "series 10 tasks", but there are cases that "series 50 tasks, 100 tasks,....". Or maybe I misunderstand your meaning.

**Comment by zycc853** [comment]
我的意思是,假如一个数据集有一百类,我们把它分为十个session,每个session有十个类,这样就持续学习了十次,.您的比较是比较了新旧任务,也就是一次session,只不过session的类别数与其他参数不同,可以这样理解吗?

**Comment by MasLiang** [comment]
So you want to compare the new classes accuracy of different session? I did a similar experiment. I used each class as the new class and compared the result.  I will add some pictures which might address your problem. I hope I get your point this time. :)

**Comment by zycc853** [comment]
是的,我想用这个方法做增量学习,假如有十个任务,每个任务代表我每次学习一个新的类,随着新类的增加,既能尽可能保持之前旧任务学习到的知识,也能学习新的类别.

**Comment by MasLiang** [comment]
I want to take an example.
First, I have a pre-trained model0.
Then I train model0 on 10 new classes and get model1.
Then I train model1 on 10 other new classes and get model2.
Then I train model2 .....
Right?
So the answer I gave yesterday is wrong. But I did a similar experiment too. I add 1 class each time. I will add a picture that might help you.

**Comment by zycc853** [comment]
是的,我就是这个意思,不过要用到LWF这个方法,请问您做的这个是用这个方法做的吗

**Comment by MasLiang** [comment]
yep~all the result is based on LwF~i.e., this code.

**Comment by MasLiang** [comment]
yep~all the result is based on LwF.

**Comment by zycc853** [comment]
好的，我会好好看的，谢谢你的回复，祝你生活愉快



---原始邮件---
发件人: "Jiawei ***@***.***&gt;
发送时间: 2021年7月16日(周五) 晚上11:19
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [MasLiang/Learning-without-Forgetting-using-Pytorch] 打扰了 (#3)




 
yep~all the result is based on LwF.
